docs/Fehlersuche.md

```diff
@@ -0,0 +1,43 @@
+#Fehlersuche
+Es kann immer mal passieren, dass etwas nicht wie gedacht funktioniert. Das kann an einem Fehler im Programmcode, an einem Hardwaredefekt oder an einer fehlerhaften oder nicht zu den Gegebenheiten passenden Konfiguration liegen.
+
+Habe ich Wenn nun eine Funktion nicht wie erwartet ausgeführt wird oder plötzlich ein Fehler auftritt, ist die erste Frage: 
+> Habe ich vor Kurzem etwas verändert?
```

Comment:
Alternativen Statzanfang in Zeile 4 nicht gelöscht?

Reply:
Ist richtig. Sollte jetzt korrigiert sein